cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DeleteLoadBalancer.java

```diff
@@ -0,0 +1,144 @@
+import io.cloudslang.content.amazon.utils.ExceptionProcessor;
+import io.cloudslang.content.amazon.utils.InputsUtil;
+import io.cloudslang.content.constants.ReturnCodes;
+
+import java.util.Map;
+
+import static io.cloudslang.content.amazon.entities.constants.Constants.Apis.LOAD_BALANCING_API;
+import static io.cloudslang.content.amazon.entities.constants.Constants.AwsParams.HTTP_CLIENT_METHOD_GET;
+import static io.cloudslang.content.amazon.entities.constants.Constants.LoadBalancingQueryApiActions.DELETE_LOAD_BALANCER;
+import static io.cloudslang.content.amazon.entities.constants.Constants.Miscellaneous.EMPTY;
+
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.ENDPOINT;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.IDENTITY;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.CREDENTIAL;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.PROXY_HOST;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.PROXY_PORT;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.PROXY_USERNAME;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.PROXY_PASSWORD;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.HEADERS;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.QUERY_PARAMS;
+import static io.cloudslang.content.amazon.entities.constants.Inputs.CommonInputs.VERSION;
+
+import static io.cloudslang.content.amazon.entities.constants.Inputs.LoadBalancerInputs.LOAD_BALANCER_ARN;
+
+import static io.cloudslang.content.constants.OutputNames.EXCEPTION;
+import static io.cloudslang.content.constants.OutputNames.RETURN_CODE;
+import static io.cloudslang.content.constants.OutputNames.RETURN_RESULT;
+
+import static io.cloudslang.content.constants.ResponseNames.FAILURE;
+import static io.cloudslang.content.constants.ResponseNames.SUCCESS;
+
+/**
+ * Created by TusaM
+ * 12/5/2016.
+ */
+public class DeleteLoadBalancer {
+    private static final String LATEST_DELETE_LOAD_BALANCER_API_VERSION = "2015-12-01";
+
+    /**
+     * Deletes the specified Application Load Balancer and its attached listeners.
+     * Note: You can't delete a load balancer if deletion protection is enabled. If the load balancer does not exist or
+     * has already been deleted, the call succeeds. Deleting a load balancer does not affect its registered targets.
+     * For example, your EC2 instances continue to run and are still registered to their target groups. If you no longer
+     * need these EC2 instances, you can stop or terminate them.
+     *
+     * @param endpoint        Optional - Endpoint to which request will be sent.
+     *                        Example: "https://elasticloadbalancing.amazonaws.com"
+     *                        Default: "https://ec2.amazonaws.com"
```

Comment:
No... the default endpoint is set to be ec2... Those op's are pointed to elasticloadbalancing ...